app.js

```diff
@@ -524,6 +524,12 @@ router.post('/account/validateSecurityCodeForTempAccount',
 const createFiatValueMiddleware = require('./middleware/fiat');
 router.get('/fiat', createFiatValueMiddleware());
 
+if (process.env.NODE_ENV === 'development') {
+    app.on(SERVER_EVENTS.SECURITY_CODE, (code) => {
+        console.log('Security code: ', code);
```

Comment:
Logging security codes to the console so it's still possible to verify a `2fa-phone` method without Twilio Verify

Reply:
Oh yeah good catch, it's actually an object 🤦🏽‍♂️ . I'll fix this up.